Source/VolunteerReporting/Web.Angular/src/app/app.module.ts

```diff
@@ -1,24 +1,37 @@
-import { CartService } from './cart/cart.service';
 import { HttpModule } from '@angular/http';
 import { FormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { Routes, RouterModule } from '@angular/router';
 
 import { AppComponent } from './app.component';
-import { CartComponent } from './cart/cart.component';
+import { CaseReportModule } from './casereport/casereport.module';
+import { CoreModule } from './core/core.module';
+
+const routes: Routes = [
+/*
```

Comment:
Commented section can be removed and instead just use the empty array directly on line 31:
`RouterModule.forRoot([]),`

Reply:
Fixed, thanks!